streamlit_folium/__init__.py

```diff
@@ -63,7 +62,7 @@ def folium_static(fig, width=700, height=500):
 
 # Create a _RELEASE constant. We'll set this to False while we're developing
 # the component, and True when we're ready to package and distribute it.
-_RELEASE = True
+_RELEASE = False
```

Comment:
Need to put this back so I can build a new PyPI package

Reply:
✅